addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -435,17 +435,19 @@
             </t>
         </template>
         <template id="price_dynamic_filter_template_product_product" name="Dynamic Product Filter Price">
-            <t t-if="data['prevent_zero_price_sale']">
-                <span t-field="website.prevent_zero_price_sale_text"/>
-            </t>
-            <t t-else="">
-                <span t-esc="data['price']"
-                      class="fw-bold"
-                      name="product_price"
-                      t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
-                <del t-if="data.get('has_discounted_price')" class="text-danger ms-1 h6" style="white-space: nowrap;"
-                     t-esc="data['list_price']"
-                     t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
+            <t t-if="not data.get('is_sample')">
+                <t t-if="data['prevent_zero_price_sale']">
+                    <span t-field="website.prevent_zero_price_sale_text"/>
+                </t>
+                <t t-else="">
```

Comment:
```suggestion
                <t t-if="data.get('prevent_zero_price_sale', False)">
                    <span t-field="website.prevent_zero_price_sale_text"/>
                </t>
                <t t-elif="'price' in data">
```

Do we really need the `is_sample` logic?

Reply:
Well, the only case when we don't fill the combination info (with the prevent_zero_price_sale key) is when we don't have a product/template, so sample data no?